tests/unit/test-class-wc-payments-token-service.php

```diff
@@ -799,4 +807,298 @@ private function generate_link_token( $stripe_id, $wp_id = 0 ) {
 		$token->save();
 		return $token;
 	}
+
+	/**
+	 * Test clear_cached_payment_methods_for_user method.
+	 */
+	public function test_clear_cached_payment_methods_for_user() {
+		$user_id     = 1;
+		$cache_key   = '_wcpay_payment_methods';
+		$cached_data = [
+			'customer_id'                          => 'cus_12345',
+			'payment_methods_woocommerce_payments' => [
+				$this->generate_card_pm_response( 'pm_test1' ),
+				$this->generate_card_pm_response( 'pm_test2' ),
+			],
+		];
+
+		// Add cached data to user meta.
+		update_user_meta( $user_id, $cache_key, $cached_data );
+
+		// Verify cached data exists.
+		$this->assertEquals( $cached_data, get_user_meta( $user_id, $cache_key, true ) );
```

Comment:
Not truly necessary. It's just testing if WordPress works which we should take for granted in this case IMHO